src/dev/src/app/tree-view/drag-and-drop-tree/drag-and-drop-tree.html

```diff
@@ -0,0 +1,13 @@
+<h4>Drag and Drop (recursive)</h4>
```

Comment:
Hi @derkoe, thank you for submitting this PR. This looks like a possible workaround to apply DnD to enable reorderable nodes in a recursive tree.

Unfortunately, I don't think we could include this PR as a feature in Clarity. First of all, we cannot recommend this API as this is adding extra div elements to place `clrDroppable`.  Also, looking at the code, it doesn't look like it followed our coding guidelines as specified in [here](https://github.com/vmware/clarity/blob/master/docs/CODING_GUIDELINES_ANGULAR.md). On top of that, as you mentioned yourself, this doesn't work reliably in some instances. To make sure its reliability and robustness, we need to write tests for it.